content/docs/adrs/angular.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Angular'
+weight: 2
+---
+
+# ADR for Angular
```

Comment:
Is this supposed to be a single ADR? It doesn't look that way.

My vision is that we create a new file for each ADR and write things in a common markdown. In the frontmatter, we put in some metadata related to category, decision, status, etc. This could be used in listings and even filters (but that's something we can figure out later). For now, we should just be able to list all the ADRs (each ADR is a separate markdown file) and sort by date rather than weight. Pages such as "Angular" and "Drupal" (assuming they are actually category of ADRs, should be rendered automatically.